db/schema.rb

```diff
@@ -291,6 +291,17 @@
     t.index ["enterprise_id", "variant_id"], name: "index_inventory_items_on_enterprise_id_and_variant_id", unique: true
   end
 
+  create_table "invoices", force: :cascade do |t|
+    t.bigint "order_id"
+    t.string "status"
+    t.integer "number"
+    t.jsonb "data"
+    t.date "date", default: -> { "CURRENT_TIMESTAMP" }
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["order_id"], name: "index_invoices_on_order_id"
+  end
+
```

Comment:
`±` Oops!
This happened because rails migrate always regenerates the schema.db file from your dev database.